kml/types.hpp

```diff
@@ -388,7 +388,7 @@ struct TrackData
 struct CategoryData
 {
   DECLARE_VISITOR_AND_DEBUG_PRINT(CategoryData, visitor(m_id, "id"),
-                                  visitor(m_compilationId, "compilationId"),
+                                  //visitor(m_compilationId, "compilationId"),
```

Comment:
So the HasCompilationsSection() should be enough, no?
Why comment here? Let's keep as-is.